### PR TITLE
133 update navbar

### LINKED ISSUE #133: Update Navbar
Update the design and functionality of the navbar.

To Do:
- [x] Add Top Navbar for Desktops
- [x] Route to Featured Page
- [x] Route to Search Page
- [x] Route to Create
- [x] Route to Profile

### PR DESCRIPTION
**Pull request by simphiwe-nonabe**
# Description

Refactored top navbar for large (Desktop) display and updated bottom navbar for small (Mobile) and medium (Tablet) display 

Fixes #133 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ryanbasiltrickett — CHANGES_REQUESTED** [review]
Add notification page route for Desktop (profile/notifications)

**Comment by PaulPilane** [comment]
Do not forget to add tests. It is failing because you did not write tests for the added functionality

File: libs/app/navigation/feature/src/navigation.component.ts
Comment (ryanbasiltrickett) [inline libs/app/navigation/feature/src/navigation.component.ts:1]: Where is the route for notifications?

File: libs/app/profile/feature/src/profile.page.ts [hunk 1]
```diff
@@ -27,6 +27,14 @@ export class ProfilePage {
     this.profile$.pipe(take(1)).subscribe(profile => this.editableProfile = Object.create(profile));
   }
 
+  handleNotificationsClicked() {
```
Comment (ryanbasiltrickett) [inline libs/app/profile/feature/src/profile.page.ts:30]: Remove these handle clicks please

**Review by ryanbasiltrickett — CHANGES_REQUESTED** [review]

**Comment by PaulPilane** [comment]
Also, pull first onto your branch and resolve conflicts

File: libs/app/profile/feature/src/profile.page.ts [hunk 2]
```diff
@@ -91,4 +99,7 @@ export class ProfilePage {
     this.closeSort();
   }
 
+  public getStore(): Store {
```
Comment (ryanbasiltrickett) [inline libs/app/profile/feature/src/profile.page.ts:102]: Please remove this, there is no need for it

**Review by ryanbasiltrickett — CHANGES_REQUESTED** [review]

File: libs/app/profile/feature/src/profile.page.ts [hunk 3]
```diff
@@ -29,6 +29,14 @@ export class ProfilePage {
     this.profile$.pipe(take(1)).subscribe(profile => this.editableProfile = Object.create(profile));
   }
 
+  // handleNotificationsClicked() {
```
Comment (ryanbasiltrickett) [inline libs/app/profile/feature/src/profile.page.ts:32]: Please remove don't comment out

File: libs/app/profile/feature/src/profile.page.spec.ts [hunk 4]
```diff
@@ -174,4 +175,62 @@ describe("ProfilePage", () => {
...
+
+  it("should open notifications page when notifications button is clicked", () => {
+    const openNotificationsSpy = jest.spyOn(page, 'openNotifications');
+    const notificationsButton = compiled.querySelector("#notifications-button");
+    notificationsButton.click();
+    expect(openNotificationsSpy).toHaveBeenCalled();
+  });
+
+  // it("should open settings page when settings button is clicked", () => {
+  //   const openSettingsSpy = jest.spyOn(page, 'openSettings');
```
Comment (ryanbasiltrickett) [inline libs/app/profile/feature/src/profile.page.spec.ts:187]: Please remove don't comment out

**Review by ryanbasiltrickett — CHANGES_REQUESTED** [review]

**Review by kphutiyagae — APPROVED** [review]

**Pull request merged**